run_scans.py

```diff
@@ -7,49 +7,98 @@ def launchTestsArgs(tool, slot, link, chamber,vt1=None,vt2=0,perchannel=False,tr
   import datetime,os,sys
   import subprocess
   from subprocess import CalledProcessError
+  from chamberInfo import chamber_config
+
+  if os.getenv('DATA_PATH') == None or os.getenv('DATA_PATH') == '':
+    print 'You must source the environment properly!'
```

Comment:
I'd also move these checks to ```__main__``` where the check on the tool is done, otherwise you'll still submit all the jobs